django_freeradius/base/models.py

```diff
@@ -91,13 +92,16 @@
                      (':=', ':='),
                      ('+=', '+='))
 
-RADCHECK_PASSWD_TYPE = ['Cleartext-Password',
-                        'NT-Password',
-                        'LM-Password',
-                        'MD5-Password',
-                        'SMD5-Password',
-                        'SSHA-Password',
-                        'Crypt-Password']
+RADCHECK_ATTRIBUTE_TYPES = ['Cleartext-Password',
```

Comment:
have you tested how this plays with the password hashing feature that is also being changed in #131?

Reply:
Let me know how I can help.  Password is definitely only one of many radcheck attribute types.

Reply:
@2stacks yes, that would be great, you could help us by testing how this patch interacts with yours. You should wait until we are ready to merge here before doing that. Hopefully this means tomorrow if @rohithasrk can make it.